packages/framework/presence/src/exposedUtilityTypes.ts

```diff
@@ -19,18 +19,16 @@ import type {
 // eslint-disable-next-line @typescript-eslint/no-namespace
 export namespace InternalUtilityTypes {
 	/**
-	 * `true` iff the given type is an acceptable shape for a notification.
+	 * `IfListener` iff the given type is an acceptable shape for a notification.
```

Comment:
I don't know how accessible `iff` is to non-English-native speakers. But we should generally avoid abbreviations.
```suggestion
	 * `IfListener` if and only if the given type is an acceptable shape for a notification.
```